src/status_im/ui/screens/profile/group_chat/views.cljs

```diff
@@ -52,17 +52,24 @@
      :action              #(re-frame/dispatch [:group-chats.ui/remove-chat-pressed chat-id])
      :accessibility-label :delete-chat-button}]))
 
-(defn member-actions [chat-id member]
+(defn member-actions [chat-id member us-admin?]
```

Comment:
is-admin?

Reply:
I though about that, but I wanted to convey the fact that it's us, not the member, and `is-admin?` might be a bit confusing, any other suggestion?

Reply:
but who is `us`?

Reply:
`us` is `us` :) I mean it's the current user (`current-public-key`) :)